types/vuelidate/lib/validators.d.ts

```diff
@@ -18,10 +18,10 @@ export interface ValidationRule {
     $pending(): boolean
 }
 
-export type CustomRule = (value: any, parentVm?: Vue) => boolean
+export type CustomRule = (value: any, parentVm?: any) => boolean
```

Comment:
what's wrong with assuming type Vue on parentVm?

Reply:
nested elements don't have a parent of Vue.

Example:

```
interface AForm {
  nested: {
    deeper: string;
  };
}

@Component({
  validations: {
    form: {
      nested: {
        required
      }
})
export class SomeComponent extends Vue {
  form: AForm = {
    nested: {
      deeper: ''
    }
  };
}
```
So Headers, the parent of`form` is in fact Vue, but the parent of`deeper` is actually `AForm['nested']` type, note Vue.  Even `nested` has a parent type of `AForm` not `Vue`